appservice/src/github/connectToGitHub.ts

```diff
@@ -98,7 +85,7 @@ async function showGitHubAuthPrompt(node: AzureTreeItem, client: SiteClient, con
     const goToPortal: vscode.MessageItem = { title: localize('goToPortal', 'Go to Portal') };
     let input: vscode.MessageItem | undefined = DialogResponses.learnMore;
     while (input === DialogResponses.learnMore) {
-        input = await vscode.window.showErrorMessage(invalidToken, goToPortal, DialogResponses.learnMore);
+        input = await vscode.window.showErrorMessage(invalidToken, { modal: true }, goToPortal, DialogResponses.learnMore);
```

Comment:
I made this a modal because the Organization quick pick menu looked like it was still loading when there's an error until you click the dialog which could be confusing.